#!/usr/bin/env python
PACKAGE = "amrl_dark_objects"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('min_area' , int_t , 0 , 'minimum area of detection' , 12 , 0 , 50 )
gen.add('crop_offset' , int_t , 0 , 'crop offset' , 10 , 0 , 100)
gen.add('min_solidity' , double_t , 0 , 'minimum solidity of detection' , .5 , 0 , 1)

exit(gen.generate(PACKAGE, "dark_node", "DarkObjects"))
